src/data/api.js

```diff
@@ -1,28 +1,23 @@
 import { getConfig } from '@edx/frontend-platform';
 import { getAuthenticatedHttpClient } from '@edx/frontend-platform/auth';
-import { logError } from '@edx/frontend-platform/logging';
 import { ExamAction } from '../constants';
 import { generateHumanizedTime } from '../helpers';
 
 const BASE_API_URL = '/api/edx_proctoring/v1/proctored_exam/attempt';
 
-async function fetchActiveAttempt() {
+async function fetchActiveAttempt(sequenceId = null) {
   // fetch 'active' (timer is running) attempt if it exists
   const activeAttemptUrl = new URL(`${getConfig().EXAMS_BASE_URL}/api/v1/exams/attempt/latest`);
   const activeAttemptResponse = await getAuthenticatedHttpClient().get(activeAttemptUrl.href);
+  if (sequenceId) {
+    // the calls the same endpoint as fetchActiveAttempt but it behaves slightly different
+    // with an exam's section specified. The attempt for that requested exam is always returned
+    // even if it is not 'active' (timer is not running)
```

Comment:
I think this comment should be updated as well, because there is only one `fetchActiveAttempt` function, so the wording here doesn't make that much sense anymore.